packages/react-static/src/utils/binHelper.js

```diff
@@ -114,8 +114,16 @@ ignoredExtensions.forEach(ext => {
   require.extensions[`.${ext}`] = () => {}
 })
 
-console.error = (err, ...rest) => {
-  console.log(new PrettyError().render(err), ...rest)
+console.error = (...args) => {
+  if(args.length === 0){
+    return console.log()
```

Comment:
```suggestion
    return undefined
```

Empty should bail out.